util/topgen/templates/xbar_tgl_excl.cfg.tpl

```diff
@@ -29,12 +29,16 @@
 %>\
 // [UNR] Exclude unused address bits based on IP address range. It is not possible to cover this.
 % for xbar in top["xbar"]:
+<%
+  # TODO: Add support for multiple ASIDs within an xbar.
```

Comment:
Is this actually needed? Otherwise remove the TODO.

Reply:
I created https://github.com/lowRISC/opentitan/issues/24801 to track that.